src/components/ui/form/select/select.tsx

```diff
@@ -47,8 +48,8 @@ const SelectInput = ({
           </span>
         </Listbox.Button>
         <Transition as={Fragment} leave="transition ease-in duration-100" leaveFrom="opacity-100" leaveTo="opacity-0">
-          <Listbox.Options className="absolute mt-1 max-h-60 w-full overflow-auto rounded-md bg-white py-1 text-base shadow-lg ring-1 ring-black ring-opacity-5 focus:outline-none sm:text-sm">
-            {data.map(e => item(e, iconCheck))}
+          <Listbox.Options className="absolute mt-1 max-h-60 w-full overflow-auto rounded-md bg-white py-1 text-base shadow-lg ring-1 ring-black ring-opacity-5 focus:outline-none max-sm:w-1/2 max-xs:w-9/12 sm:text-sm">
```

Comment:
What are `max-xs` and `max-sm`? Is some custom Tailwind styles?

Reply:
it's one of feature tailwind for design responsive, [documentation](https://tailwindcss.com/docs/responsive-design#targeting-a-breakpoint-range)

Reply:
I see. I just found out.